app/scripts/acs/acs-graphs.js

```diff
@@ -61,27 +61,27 @@
                     chart = nv.models.multiBarHorizontalChart()
                         .x(function(d) { return addElipses(d.label, opts.labelCharacters); })
                         .y(function(d) { return d.value; })
-                        .tooltipContent(function(d) {
-                            var percent = numberFilter(d.data.value / total * 100, 1);
-                            return ['<b>', d.data.label, ': </b>', numberFilter(d.data.value),
-                                    ' (', percent, '%)'].join('');
-                        })
                         .color(function () { return '#bc5405'; })
                         .margin(opts.margin)
                         .showControls(false)
                         .showLegend(false);
+                    chart.tooltip.contentGenerator(function (d) {
+                        var percent = numberFilter(d.data.value / total * 100, 1);
+                        return ['<b>', d.data.label, ': </b>', numberFilter(d.data.value),
+                                ' (', percent, '%)'].join('');
+                    });
                 } else {
                     chart = nv.models.discreteBarChart()
                         .x(function(d) { return addElipses(d.label, opts.labelCharacters); })
                         .y(function(d) { return d.value; })
-                        .tooltipContent(function(d) {
-                            var percent = numberFilter(d.data.value / total * 100, 1);
-                            return ['<b>', d.data.label, ': </b>', numberFilter(d.data.value),
-                                    ' (', percent, '%)'].join('');
-                        })
                         .staggerLabels(true)
                         .color(function () { return '#bc5405'; })
                         .margin(opts.margin);
+                    chart.tooltip.contentGenerator(function (d) {
+                        var percent = numberFilter(d.data.value / total * 100, 1);
+                        return ['<b>', d.data.label, ': </b>', numberFilter(d.data.value),
+                                ' (', percent, '%)'].join('');
+                    });
```

Comment:
These are identical, right?  Moving them right below the conditional would work the same and only require one copy.